packages/storage/src/common/urls.ts

```diff
@@ -39,6 +40,10 @@ export const DEFAULT_GATEWAY_URLS: GatewayUrls = {
  */
 export const TW_UPLOAD_SERVER_URL = "https://storage.thirdweb.com";
 
+export const CUSTOM_UPLOAD_SERVER_URL = getProcessEnv(
+  "CUSTOM_UPLOAD_SERVER_URL",
+);
```

Comment:
why not do the defaultValue strategy here, why does the underlying consumer need to know about 2 values?

```js
export const TW_UPLOAD_SERVER_URL = getProcessEnv(
  "CUSTOM_UPLOAD_SERVER_URL",
"https://storage.thirdweb.com");
```

is kinda what I was thinking?

Reply:
I thought about it but we need to know if there's a custom url or not because we shouldn't pass the clientId/secretKey in case there's a custom url